packages/drivers/file-driver/src/fileDocumentStorageService.ts

```diff
@@ -93,23 +93,29 @@ export class FluidFetchReader extends ReadDocumentStorageServiceBase implements
      * Finds if a file exists and returns the contents of the blob file.
      * @param sha - Name of the file to be read for blobs.
      */
-    public async read(sha: string): Promise<string> {
+    public async readBlob(sha: string): Promise<ArrayBufferLike> {
         if (this.versionName !== undefined) {
-            const fileName = `${this.path}/${this.versionName}/${sha}`;
+            const fileName = `${this.path}/${this.versionName}/decoded/${sha}`;
             if (fs.existsSync(fileName)) {
-                const data = fs.readFileSync(fileName).toString();
+                const data = fs.readFileSync(fileName);
                 return data;
             }
         }
         throw new Error(`Can't find blob ${sha}`);
     }
 
     /**
-     * {@inheritDoc @fluidframework/driver-definitions#IDocumentStorageService.readString}
+     * @deprecated - here for maintaining perf, will be removed after storage returns binary data
      */
-    public async readString(sha: string): Promise<string> {
```

Comment:
Do we need to keep readString for back compat?

Reply:
same comment for other places where readString implementation is removed.

Reply:
maybe not? cause readString is never called anywhere